lib/mbimport.js

```diff
@@ -223,7 +223,8 @@ var MBImport = (function () {
                     total_duration += duration_ms;
                 }
                 appendParameter(parameters, `mediums.${i}.track.${j}.length`, tracklength);
-
+                appendParameter(parameters, `mediums.${i}.track.${j}.recording`,
+                    track.recording ? track.recording : '');
```

Comment:
The `appendParameter` function already checks if the input is not empty:
```suggestion
                appendParameter(parameters, `mediums.${i}.track.${j}.recording`, track.recording);
```